File: ExamPreparation/sebi/web practical/csharp/flightshotelsreservations/backend/Model/Hotel.cs
public class Hotel
{
    public int HotelID { get; set; }
    public string HotelName { get; set; } = "";
    public string City { get; set; } = "";
    public DateOnly Date { get; set; } = DateOnly.FromDateTime(DateTime.Now);
    public int AvailableRooms { get; set; }
}

File: ExamPreparation/Flightssss/lab9/src/main/java/com/example/lab9/model/Hotel.java
package com.example.lab9.model;

import java.time.LocalDate;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "Hotels")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Hotel {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(length = 128, nullable = false)
    private String name;

    @Column(nullable = false)
    private LocalDate date;

    @Column(nullable = false)
    private String city;

    @Column(nullable = false)
    private Integer availableRooms;
}
